samples/nrf9160/lwm2m_client/src/main.c

```diff
@@ -73,6 +73,66 @@ void ncell_meas_work_handler(struct k_work *work)
 }
 #endif
 
+
+#if defined(CONFIG_APP_LWM2M_CONFORMANCE_TESTING)
+static struct k_work_delayable send_periodical_work;
+static uint8_t send_count = 0;
+
+static int server_send_mute_cb(uint16_t obj_inst_id, uint16_t res_id, uint16_t res_inst_id, uint8_t *data,
+			uint16_t data_len, bool last_block, size_t total_size)
+{
+	if (*data) {
+		LOG_ERR("Server Muted Send");
+	} else {
+
+		if (send_count == 0) {
+			LOG_ERR("Server Activate Send");
+			send_count = 5;
+		}
+		k_work_cancel_delayable(&send_periodical_work);
+		k_work_schedule(&send_periodical_work, K_SECONDS(1));
```

Comment:
What is the purpose of cancel when you immediately schedule after that?

Why do you need to touch the schedule at all?

Reply:
Ok. I change that I keep started shedule when TX is completed.